docs/policy/collab-bearer-tokens.md

```diff
@@ -79,6 +80,35 @@ Scopes utilized by OSG services include the following:
 
 Replacing `<PATH>` with a path to the storage location that the bearer should be authorized to access.
 
+### Issuer ###
+
+The issuer URL, or the `iss` claim, indicates the endpoint to use for authenticating a given token.
+A collaboration may have more than one token issuer but a single token issuer should never serve more than one
+collaboration.
+In other words, given the token issuer, the site can determine the collaboration that issued the token.
+The following collaborations have registered token issuers with the OSG:
+
+| **Collaboration** | **Issuers**                                                                                   |   |
+|-------------------|-----------------------------------------------------------------------------------------------|---|
+| ATLAS             | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/ATLAS.yaml>    |   |
+| CLAS12            | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/CLAS12.yaml>   |   |
+| CMS               | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/CMS.yaml>      |   |
+| DES               | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/DES.yaml>      |   |
+| DUNE              | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/DUNE.yaml>     |   |
+| EIC               | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/EIC.yaml>      |   |
+| Fermilab          | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/Fermilab.yaml> |   |
+| CHTC (née GLOW)   | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/GLOW.yaml>     |   |
+| Gluex             | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/Gluex.yaml>    |   |
+| IceCube           | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/IceCube.yaml>  |   |
+| LIGO              | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/LIGO.yaml>     |   |
+| Mu2e              | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/Mu2e.yaml>     |   |
+| OSPool            | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/OSG.yaml>      |   |
+| SBND              | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/SBND.yaml>     |   |
+| gm2               | <https://github.com/opensciencegrid/topology/blob/master/virtual-organizations/gm2.yaml>      |   |
+
+The OSG distributes the `osg-scitokens-mapfile` RPM package that includes default issuer and [subject](#subject)
```

Comment:
I think one of the "default"s is redundant...

Reply:
Good catch